### PR TITLE
Fix: #980. Clarify binary payload is b64 encoded

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

clarify editorial rendering of binary payloads

### CODE REVIEW

File: draft-ietf-httpbis-digest-headers.md [hunk 1]
```diff
@@ -803,6 +807,8 @@ Digest: sha-256=4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=, id-sha-256=X48E9qO
 iwiAeyJoZWxsbyI6ICJ3b3JsZCJ9Aw==
 ~~~
 
+The above payload body contains non-printable characters and it is shown as base64 encoded.
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md]: same as above comment, lets add this to the response description at the start of section. e.g.

```
The response includes a `Content-Encoding: br` that indicates the selected representation is brotli encoded, the payload body is displayed as base64. There are two `Digest` field-values:

<bullets as they are> 
```

File: draft-ietf-httpbis-digest-headers.md [hunk 2]
```diff
@@ -740,6 +742,8 @@ Digest: sha-256=4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=
...
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md]: Lets add this to the response description at the start of section. e.g.

```
The response includes a `Content-Encoding: br` that indicates the selected representation is brotli encoded, the payload body is displayed as base64.
```

File: draft-ietf-httpbis-digest-headers.md [hunk 3]
```diff
@@ -233,6 +233,8 @@ the intermediaries may apply.
 
 The following examples show how representation metadata, payload transformations and method
 impacts on the message and payload body.
+When the payload body contains non-printable characters (eg. when it is compressed)
+it is shown as base64 encoded.
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md]:
```suggestion
it is shown as a base64-encoded string.
```
maybe better?

File: draft-ietf-httpbis-digest-headers.md [hunk 4]
```diff
@@ -718,6 +720,9 @@ The response includes a `Content-Encoding: br` that indicates
 the selected representation is brotli encoded.
 The `Digest` field-value is therefore different compared to the request.
 
+The response body is displayed as a base64-encoded string
```
Comment (ioggstream) [inline draft-ietf-httpbis-digest-headers.md]: I prefer to separate the sentence because of the `therefore` in line 721. @LPardue

Comment (LPardue) [reply draft-ietf-httpbis-digest-headers.md]: I can live with the sentence order but there is no need to split the paragraph IMO.

This is an example where I think the general document readability can be improved with another editorial pass over all examples. However, we can tackle that in a separate PR once design issues are closed out.

Comment (ioggstream) [reply draft-ietf-httpbis-digest-headers.md]: Sure! We can schedule it one month before Vancouver: wdyt?

Comment (LPardue) [reply draft-ietf-httpbis-digest-headers.md]: SGTM

**Pull request merged**